fastlane/docs/README.md

```diff
@@ -54,6 +64,18 @@ after_all do |lane|
 end
 ```
 
+### `after_each` block
+
+This block will get executed *after* each lane switch (and before returning to a parent lane). It supports the same actions as lanes.
```

Comment:
`get executed *after* each lane switch`

`and before returning to a parent lane`

I don't understand the difference, can we make the docs more clear?